packages/webapp/src/components/Task/AssignTask/useTaskAssignForm.jsx

```diff
@@ -109,7 +111,8 @@ const useTaskAssignForm = ({
   const hourlyWage = watch(HOURLY_WAGE);
 
   const disabled = !selectedWorker || !isValid;
-  const assigned = selectedWorker.label !== unAssignedOption.label;
+  const assigned =
+    isAssigned === undefined ? selectedWorker.label !== unAssignedOption.label : isAssigned;
```

Comment:
Does `assigned` need to be false when a user is selected by default??

Reply:
The thing here is: when I pre-fill the input with the only user then assigned becomes true, just think this is not the right way.

Reply:
Oh sorry, I guess I should have given the variable a better name. `assigned` actually means "user is being selected", so when there is a default assignee, we still want `assigned` to be true.

Reply:
So isAssigned can be removed and the original line should just work, right?

Reply:
Yes, that's correct!